src/components/atoms/Button/Button.tsx

```diff
@@ -92,10 +92,15 @@ const StyledButton = styled.button`
     background-color: var(--color-black);
   }
 
+  &.white {
+    background-color: var(--color-white);
+    color: var(--color-black);
+  }
+
   &:hover {
     background-color: var(--color-primary-light);
     color: var(--color-white-fix);
-    box-shadow: var(--shadow);
+    box-shadow: 0 0 6px var(--color-light-10);
```

Comment:
should this be globally available or only for this component?

Reply:
just for this component